.pre-commit-config.yaml

```diff
@@ -75,7 +76,7 @@ repos:
         exclude: docs/source-app
 
   - repo: https://github.com/psf/black
-    rev: 22.6.0
+    rev: 22.10.0
```

Comment:
You could update the rest of the hooks and leave black out for now, as that's the one that requires more work.